assets/src/components/parts/janus-capi-iframe/ExternalActivity.tsx

```diff
@@ -735,7 +735,7 @@ const ExternalActivity: React.FC<PartComponentProps<CapiIframeModel>> = (props)
         value = JSON.stringify(val);
       }
       response.values.responseType = 'success';
-      response.values.value = value?.length ? value : '{}';
+      response.values.value = value?.length ? value : '[]';
```

Comment:
Previously I made '{}' in PR (https://github.com/Simon-Initiative/oli-torus/pull/2502) that fixed 'Add Observation' button not working. Making it '[]', fixes both the issues. The SIMs search for key-values in value field of the GET_DATA_RESPONSE so when we used to send '' or '{}', it throws error. In this issue, 'https://reflector.argos.education/reflector/sim/spr-widget-image-evidence/prod/1.*' widget was throwing console error.